engines/director/transitions.cpp

```diff
@@ -821,7 +829,9 @@ void Window::dissolvePatternsTrans(TransParams &t, Common::Rect &clipRect, Graph
 			break;
 		}
 
-		g_system->delayMillis(t.stepDuration);
+		uint32 endTime = g_system->getMillis(true);
```

Comment:
Why are you asking to skip Event Recorder here, e.g. `true` parameter? It should be used with `true` only within the event recorder itself

Reply:
Oops, fixed in all the spots.